### PR TITLE
Release 3.5 Changelog

### PR DESCRIPTION
**Pull request by Guaris**
https://github.com/Kong/kong-ee/blob/next/3.5.x.x/changelog/3.5.0.0/3.5.0.0.md
https://github.com/Kong/kong-ee/blob/next/3.5.x.x/changelog/3.5.0/3.5.0.md

### CODE REVIEW

File: app/gateway/changelog.md [hunk 1]
```diff
@@ -10,11 +10,245 @@ Changelog for supported Kong Gateway versions.
...
+
+### Fixes
+
+#### Enterprise
+
+* Fixed a keyring issue where Kong nodes failed to send keyring material when using the cluster strategy.
+* Enforced Content Security Policy (CSP) headers for serving static resources via Dev Portal and Kong Manager.
+* Fixed RBAC issue related to retrieving group roles with a numeric group name type.
+* Added support for specifying Azure Vault configuration options in the kong.conf file.
+* Hardcoded `admin_gui_auth_conf` settings for parts while using openid-connect as the `admin_gui_auth` method.
```
Comment (cloudjumpercat) [inline app/gateway/changelog.md:120]: I'm not sure what "for parts" means in this context.

File: app/gateway/changelog.md [hunk 2]
```diff
@@ -10,11 +10,245 @@ Changelog for supported Kong Gateway versions.
...
+* Fixed an issue causing upstream SSL failures when plugins use response handlers. [#11502](https://github.com/Kong/kong/issues/11502)
+* Corrected an issue where the `tls_passthrough` protocol could not work with expressions flavor. [#11538](https://github.com/kong/kong/issues/11538)
+* Fixed a bug that caused a failure in sending tracing data to Datadog when the value of the x-datadog-parent-id header in requests was a short decimal string. [#11599](https://github.com/kong/kong/issues/11599).
+* Resolved the building failure when applying patches. [#11696](https://github.com/kong/kong/issues/11696).
+* Enabled the use of vault references in DBless mode in declartive configs. [#11845](https://github.com/kong/kong/issues/11845)
+* Properly warmed up vault caches during initalization. [#11827](https://github.com/kong/kong/issues/11827).
+* Respected Vault resurrect time in case a vault secret is deleted from a vault. [#11852](https://github.com/kong/kong/issues/11852).
+* Restore lapis and `luarocks-admin` bins [#11551](https://github.com/Kong/kong/pull/11551).
```
Comment (cloudjumpercat) [inline app/gateway/changelog.md:169]:
```suggestion
* Restored lapis and `luarocks-admin` bins [#11551](https://github.com/Kong/kong/pull/11551).
```

**Review by cloudjumpercat — COMMENTED** [review]
Just left a few nitpicks! Looks good!

File: app/_src/gateway/breaking-changes/35x.md [hunk 3]
```diff
@@ -8,3 +8,20 @@ affect your current installation.
 
 You may need to adopt different upgrade paths depending on your deployment methods, set of features in use,
 custom plugins, for example.
+
+## Dev Portal and Vitals
+
+As of this release, the product component known as Kong Enterprise Portal is no longer included in the Kong Gateway Enterprise (previously known as Kong Enterprise) software package. Existing customers who have purchased Kong Enterprise Portal can continue to use it and be supported via a dedicated mechanism.
```
[the remark this answers is not in the document]
Comment (lena-larionova) [reply app/_src/gateway/breaking-changes/35x.md:14]: Leaving as-is. This is the "official" (ie, with legal considerations) language from Tom.

**Comment by lena-larionova** [comment]
There is one remaining 404, but that'll go away once https://github.com/Kong/docs.konghq.com/pull/6475 is merged.

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**